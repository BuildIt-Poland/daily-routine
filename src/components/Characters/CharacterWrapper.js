import styled from 'styled-components';

import media from '../../styles/media';
import { spacingLarge } from '../../styles/designTokens';

const CharacterWrapper = styled.div`
  padding: 0;
  padding-bottom: ${spacingLarge};
  position: relative;

  ${media.phone`
    padding-bottom: 0;
  `};
`;

export default CharacterWrapper;
