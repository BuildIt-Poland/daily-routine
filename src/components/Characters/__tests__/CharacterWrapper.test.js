import React from 'react';
import { create } from 'react-test-renderer';

import CharacterWrapper from '../CharacterWrapper';

describe('COMPONENT - Characters CharacterWrapper', () => {
  it('renders correctly', () => {
    const component = create(<CharacterWrapper />);

    expect(component.toJSON()).toMatchSnapshot();
  });
});
