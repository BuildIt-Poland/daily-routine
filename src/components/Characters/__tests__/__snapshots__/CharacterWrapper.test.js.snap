// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`COMPONENT - Characters CharacterWrapper renders correctly 1`] = `
.c0 {
  padding: 0;
  padding-bottom: 3rem;
  position: relative;
}

@media (max-width:26.25em) {
  .c0 {
    padding-bottom: 0;
  }
}

<div
  className="c0"
/>
`;
